pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -35,7 +35,7 @@ let
 
   mkUniquePkgs = pkgs: fastUnique (a: b: a < b) # highlighting hack: >
     # here we deal with those dummy packages needed for hyphenation filtering
-    (map (p: if lib.isDerivation p then builtins.toPath p else "") pkgs);
+    (map (p: if lib.isDerivation p then p.outPath else "") pkgs);
```

Comment:
@vcunat Can you validate that this is the right thing here?

Reply:
It needs something that "gets converted" to the store path and is reasonably ordered by `<` operator.  I think `.outPath` should work.